modules/gitops-release-notes-1-18-2.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-18.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-18-2_{context}"]
+= Release notes for {gitops-title} 1.18.2
+
+{gitops-title} 1.18.2 is available on {OCP} 4.14, 4.16, 4.17, 4.18, 4.19, and 4.20.
+
+[IMPORTANT]
+====
+After completing the upgrade, run the `audit-operator-roles.sh` audit script to review namespace-scoped access created by the {gitops-title} Operator. The script identifies `Role` and `RoleBinding` objects that grant cross-namespace access when the _Applications in any namespace_ feature or _ApplicationSets in any namespace_ feature is enabled using the Argo CD custom resource (CR) (`.spec.sourceNamespaces` and `.spec.applicationSet.sourceNamespaces`). These features are disabled by default. The script scans all namespaces and reports `Role`/`RoleBinding` combinations that meet the following conditions:
+
+* Grant access to the `argoproj.io/Application` resource
+* Are labeled with `app.kubernetes.io/part-of=argocd`
+* Bind to a service account in a different namespace, resulting in cross-namespace access
+
+Review the script output to ensure that cross-namespace access is limited to the intended namespaces only.
+
+For more information about the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/audit-operator-roles.sh[Audit Operator script].
+
+For more information about the steps to execute the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/README.md[Audit Operator script readme].
+====
+
+[id="errata-updates-1-18-2_{context}"]
+== Errata updates
+
+[id="RHSA-2025:23203-gitops-1-18-2-security-update-advisory_{context}"]
+=== RHSA-2025:23203 - {gitops-title} 1.18.2 security update advisory
+
+Issued: 2025-12-15
+
+The list of security fixes included in this release is available in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2025:23203[RHSA-2025:23203]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-18-2_{context}"]
+== Fixed issues
+
+Progressive Sync status and UI stability improvements::
+With this update, multiple issues are resolved where applications generated by `ApplicationSets` incorrectly displayed the Progressive Sync status as `Unknown` in the UI. The Progressive Sync state is now displayed correctly, and when the feature is disabled, the UI hides the field entirely. This update also fixes UI crashes that occur when Progressive Sync was enabled but the `applicationStatus` field is not available in the `ApplicationSet` status.
```

Comment:
Consider following the language guidelines to write Fixed issues release notes, i.e.

```
Heading that summarizes the fixed issue
Before this update, <cause>. As a consequence, <consequence>. With this release, <fix>. As a result, <result>.

TICKET-REFERENCE
```
Reference: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-fixed-issues

Consider doing this for all the fixed issues release notes mentioned below.